### ISSUE TITLE
1.6.0.1 - Local Cordova Plugin Dev -- Plugin "XXX" is not present in the project

### ISSUE THREAD

**Issue by tripflex** [meteor/meteor#9561]
I'm working on updating a forked cordova plugin (WifiWizard2) from my local filesystem, and everytime I run `meteor run-android-device` after the first initial cordova build, I keep getting the error 

```shell
Cordova error: Plugin "wifiwizard2" is not present in the project
```

Verbose Output:

```shell
Reinstalling Cordova plugins added from the local path
Removing plugins cordova-plugin-meteor-webapp,cordova-plugin-splashscreen,cordova-plugin-statusbar,cordova-plugin-whitelist,cordova-plugin-wkwebview-engine,es6-promise-plugin,wifiwizard2 from Cordova project
%% No scripts found for hook "before_plugin_rm".
%% Calling plugman.uninstall on plugin "cordova-plugin-meteor-webapp" for platform "android"
%% Calling plugman.uninstall on plugin "cordova-plugin-splashscreen" for platform "android"
%% Calling plugman.uninstall on plugin "cordova-plugin-statusbar" for platform "android"
%% Calling plugman.uninstall on plugin "cordova-plugin-whitelist" for platform "android"
%% Calling plugman.uninstall on plugin "cordova-plugin-wkwebview-engine" for platform "android"
%% Calling plugman.uninstall on plugin "es6-promise-plugin" for platform "android"
%% Running command: /Users/myles/cloud/dev/github/WifiWizard2Demo/src/.meteor/local/cordova-build/platforms/android/cordova/version
%% Running command: /Users/myles/cloud/dev/github/WifiWizard2Demo/src/.meteor/local/cordova-build/platforms/android/cordova/version
%% Running command: /Users/myles/cloud/dev/github/WifiWizard2Demo/src/.meteor/local/cordova-build/platforms/android/cordova/version
%% Running command: /Users/myles/cloud/dev/github/WifiWizard2Demo/src/.meteor/local/cordova-build/platforms/android/cordova/version
%% Running command: /Users/myles/cloud/dev/github/WifiWizard2Demo/src/.meteor/local/cordova-build/platforms/android/cordova/version
%% Running command: /Users/myles/cloud/dev/github/WifiWizard2Demo/src/.meteor/local/cordova-build/platforms/android/cordova/version
=> Errors executing Cordova commands:

   While removing plugins cordova-plugin-meteor-webapp,cordova-plugin-splashscreen,cordova-plugin-statusbar,cordova-plugin-whitelist,cordova-plugin-wkwebview-engine,es6-promise-plugin,wifiwizard2 from Cordova project:
   Cordova error: Plugin "wifiwizard2" is not present in the project. See `cordova plugin list`.
   at /Users/myles/.meteor/packages/meteor-tool/.1.6.0_1.1u0toj2.l6xqf++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/cordova-lib/src/cordova/plugin.js:286:45
```

This is a bare meteor project with the following file structure:

```shell
/WifiWizard2Demo/src/
/WifiWizard2Demo/plugins/WifiWizard2
```

With all meteor files under the `src` directory.

The only way to currently resolve this is to remove cordova build each time before calling `meteor run android-device`

```shell
rm -rf .meteor/local/cordova-build
```

Is this a known issue that maybe I wasn't able to find in any open issues?  This only happens for local dev plugins.  I created this demo bare project specifically for testing purposes so it's basically a bare project.

My assumption is that during the cordova build since it tries to uninstall and then reinstall that somewhere along the line there is an issue?

If I run `cordova plugin list` from `cordova-build` directory, it does show the `wifiwizard2` plugin, but for some reason during uninstall process it isn't detected?  Maybe it's because Meteor creates a symlink?

**Comment by hwillson** [comment]
> Is this a known issue that maybe I wasn't able to find in any open issues?

It is known (see the very similar https://github.com/meteor/meteor/issues/9548), but let's keep this issue open since it's slightly different. There are plans to address this in Meteor 1.6.2 (see https://github.com/meteor/meteor/issues/9548#issuecomment-359200660), so your issue details will help. Thanks!

**Comment by tripflex** [comment]
@hwillson thanks for the reply, let me know if there's anything you guys need from my end for testing or any other debug info, thanks!

**Comment by tripflex** [comment]
@hwillson this appears to no longer be an issue with the last release of Meteor 1.6.1 (at least for me, it is no longer a problem)

**Comment by hwillson** [comment]
Closing for now as this appears to be fixed with 1.6.1 - thanks!
